src/main/java/marquez/api/models/NamespaceResponse.java

```diff
@@ -14,16 +14,32 @@
 
 package marquez.api.models;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import static marquez.common.base.MorePreconditions.checkNotBlank;
 
-@Data
-@NoArgsConstructor
-@AllArgsConstructor
+import java.util.Optional;
+import javax.annotation.Nullable;
+import lombok.*;
```

Comment:
_Minor:_ This would fail our style guide, see https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
sure, will make note of it

Reply:
will make updates to other classes as well, where applicable

Reply:
change has been implemented per review comment